packages/commercetools/composables/nuxt/helpers/index.js

```diff
@@ -1,4 +1,6 @@
 import defaultConfig from '@vue-storefront/commercetools/nuxt/defaultConfig';
+import { createIntegrationPlugin } from '@vue-storefront/core';
```

Comment:
'core/nuxt` or `@vue-storefront/nuxt` ?

Reply:
didn't work :/ but i need to work on this